client/src/Plant/PlantForm.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+
+import Api from "../Api";
+
+function PlantForm(){
```

Comment:
Plant SINGULAR is CORRECT in this component. Because this form component only edits a SINGLE instance of a Plant.

Does this start to help make sense how the singular/plural naming conventions work?